atom/instance.py

```diff
@@ -15,12 +15,13 @@ class Instance(Member):
     This call is equivalent to `isinstance(value, kind)` and all the
     same rules apply.
 
-    The value of an Instance may be set to None.
+    The value of an Instance may be set to None  if optional is True,
+    otherwise None won't be considered as a valid value.
```

Comment:
`If optional is True, the value of an Instance may be set to None,  otherwise None is not considered as a valid value.`

Maybe?

Reply:
Thanks